app/assets/stylesheets/layout.scss

```diff
@@ -17,3 +16,9 @@ body {
     margin-right: $nprogress-spinner-icon-width;
   }
 }
+
+textarea.code,
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `ace_editor` should be written in lowercase with hyphens